anitya/db/models.py

```diff
@@ -954,6 +954,31 @@ def get_id(self):
         """
         return six.text_type(self.id)
 
+    def __json__(self):
```

Comment:
I know this is copied from other model, but I do not think the name of this method conveys what it does. It's not a "real" dunder method, nor is it actually returning JSON. ``as_dict`` or ``to_dict`` or something is clearer.

Reply:
Yeah, you are right.